testnets/mantrachaintestnet/assetlist.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "mantrachaintestnet",
+  "assets": [
+    {
+      "description": "The native token of MANTRA Chain",
+      "denom_units": [
+        {
+          "denom": "uaum",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "aum",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uaum",
+      "name": "MANTRA Chain",
+      "display": "aum",
+      "symbol": "AUM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mantra/images/mantra.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mantra/images/mantra.svg"
+      },
+      "coingecko_id": "",
+      "keywords": [
+        "mantra",
+        "staking",
+        "delegating",
+        "governance",
+        "regulation",
+        "defi"
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mantra/images/mantra.png",
```

Comment:
For all image references, please update the path to point to the ...master/mantratestnet/images/ directory, since master/mantra/ doesn't exist.

Reply:
Updated all image references to: `"https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/mantrachaintestnet/images/...`